src/backend/marsha/core/tests/test_views_lti_video.py

```diff
@@ -1257,6 +1257,7 @@ def test_views_lti_video_post_error(self, mock_verify, mock_logger):
         self.assertEqual(
             context,
             {
+                "appName": "",
```

Comment:
Is it possible to add a test with `appName` not empty ?

Reply:
I don't know if we can simulate an app, but a test with filled appName will come with [BBB app](https://github.com/openfun/marsha/pull/1107/) (src/backend/marsha/bbb/tests/test_views_lti.py)
Do you thing it is enough ?

Reply:
Yes